sdk/Javascript/src/MobileServiceSyncTable.js

```diff
@@ -0,0 +1,68 @@
+﻿// ----------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// ----------------------------------------------------------------------------
+
+/// <reference path="C:\Program Files (x86)\Microsoft SDKs\Windows\v8.0\ExtensionSDKs\Microsoft.WinJS.1.0\1.0\DesignTime\CommonConfiguration\Neutral\Microsoft.WinJS.1.0\js\base.js" />
+/// <reference path="C:\Program Files (x86)\Microsoft SDKs\Windows\v8.0\ExtensionSDKs\Microsoft.WinJS.1.0\1.0\DesignTime\CommonConfiguration\Neutral\Microsoft.WinJS.1.0\js\ui.js" />
+/// <reference path="Generated\MobileServices.DevIntellisense.js" />
+
+var Validate = require('Validate');
+var Platform = require('Platform');
+
+function MobileServiceSyncTable(tableName, client) {
+    /// <summary>
+    /// Creates an instance of the MobileServiceSyncTable class.
+    /// </summary>
+    /// <param name="tableName" type="String">
+    /// Name of the table.
+    /// </param>
+    /// <param name="client" type="MobileServiceClient" mayBeNull="false">
+    /// The MobileServiceClient used to make requests.
+    /// </param>
+
+    this.getTableName = function () {
+        /// <summary>
+        /// Gets the name of the table.
+        /// </summary>
+        /// <returns type="String">The name of the table.</returns>
+
+        return tableName;
+    };
+
+    this.getMobileServiceClient = function () {
```

Comment:
nit: since "Service" will be a bad name long term, let's try just getClient here.